src/main/java/org/jenkinsci/plugins/github/GitHubPlugin.java

```diff
@@ -41,7 +45,6 @@ public static void runMigrator() throws Exception {
 
     @Override
     public void start() throws Exception {
-        addXStreamAliases();
```

Comment:
shouldn't you remove this method completely now?

Reply:
I could